### ISSUE TITLE
DOM only updates if BrowserWindow is visible

### ISSUE THREAD

**Issue by mulhoon** [electron/electron#5562]
* Electron version: 1.0.0
* Operating system: OSX

I suspect this might be impossible but, I have a small tray BrowserWindow and a large BrowserWindow. Only one can be shown at a time, you can toggle between. Changes to the DOM in one window get passed to the other (via ipc) to keep them both in sync. The problem comes that the dom only seems to visually change when ```win.show()``` is called, so you see a blink when toggling between the windows while the dom updates.

Note: I have two windows (instead of resizing the same window) because the views are very different.

**TL:DR** Is there a way to force the rendering of dom on a Browser Window that isn't visible?

**Comment by felixrieseberg** [comment]
You'll need to disable `backgroundThrottling` on the `webPreferences` of your browser window. See [the docs](https://github.com/electron/electron/blob/master/docs/api/browser-window.md) for more info!

**Comment by mulhoon** [comment]
You're my hero. Thank you.

```
webPreferences:{
	backgroundThrottling:false
}
```
Bingo!
